osmaxx/excerptexport/static/excerptexport/scripts/maps/excerpt_viewer.js

```diff
@@ -18,39 +18,17 @@ window.ExcerptViewer = function(mapElementID, excerptApiUrl) {
         this.map.addLayer(layer);
     }.bind(this);
 
-    this._extendBounds = function(bounds, marginFactor) {
-        var latDiff = bounds._northEast.lat - bounds._southWest.lat;
-        var lonDiff = bounds._northEast.lon - bounds._southWest.lon;
-        bounds._northEast.lat += latDiff*marginFactor;
-        bounds._southWest.lat -= latDiff*marginFactor;
-        bounds._northEast.lon += lonDiff*marginFactor;
-        bounds._southWest.lon -= lonDiff*marginFactor;
-        return bounds;
-    };
-
-    this.showExcerptOrCountryOnMap = function(ID) {
+    this.showExcerptOnMap = function(ID) {
         L.geoJson.ajax(this.excerptApiUrl.replace('{ID}', ID)).on('data:loaded', function(event) {
             // We are certain that there is only one layer on this feature, because our API provides it so.
-            var feature_type = event.target.getLayers()[0].feature.properties.type_of_geometry;
+            var color = event.target.getLayers()[0].feature.properties.color || 'black';
```

Comment:
or use a completely distinct color (red?), so that we'll notice the unexpected bahavior